src/Main.ts

```diff
@@ -1499,4 +1552,30 @@ export class Main {
     private onReadyProbe(_, res: Response) {
         res.status(this.ready ? 201 : 425).send("");
     }
+
+    private async onUserActivityChanged(state: UserActivityState) {
+        for (const userId of state.changed) {
+            await this.datastore.storeUserActivity(userId, state.dataSet.users[userId]);
+        }
+        this.bridgeBlocker?.checkLimits(state.activeUsers);
+    }
+
+    async disableHookHandler() {
+        await this.slackHookHandler?.close().then(() => log.info("Disabled hook handler"));
+    }
+
+    enableHookHandler() {
+        this.slackHookHandler = new SlackHookHandler(this);
+        log.info("Enabled hook handler");
+    }
+
+    async disableRtm() {
+        await this.slackRtm?.disconnectAll().then(() => log.info("Disabled RTM"));
```

Comment:
Ooi, why the `then`? This should be identical?
```suggestion
        await this.slackRtm?.disconnectAll();
        log.info("Disabled RTM");
```

Reply:
The alternative would fire the log.info() even if slackRtm is undefined and thus not really disabled. It does look confusing though, an `if` may be less concise but is probably more obvious. Will change it.

Reply:
Done in bc8a22e